src/views/Swap.vue

```diff
@@ -237,7 +235,7 @@ export default {
     amountError () {
       const amount = BN(this.safeAmount)
 
-      if (amount.gt(this.available)) return 'Amount exceeds available balance.'
+      if (amount.gt(this.available)) return 'Lower amount. This exceeds your available balance.'
```

Comment:
Seems to drop into 2 lines :(

Reply:
Agree, let me as to Thessy about it

Reply:
In this case I need to put more to the left the label to use the message that the want to use